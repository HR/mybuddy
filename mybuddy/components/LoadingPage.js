import React, { Component } from 'react'
import { StyleSheet } from 'react-native'
import { Container, Spinner } from 'native-base'

import DEFAULT_COLORS from '../constants/Colors'

export default class LoadingPage extends Component {
  render() {
    return (
      <Container style={styles.container}>
        <Spinner color={DEFAULT_COLORS.secondaryColor} />
      </Container>
    )
  }
}

const styles = StyleSheet.create({
  container: {
    justifyContent: 'center'
  }
})
