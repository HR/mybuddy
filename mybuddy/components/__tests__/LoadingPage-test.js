import 'react-native';
import React from 'react';
import LoadingPage from '../LoadingPage';
import renderer from 'react-test-renderer';

describe('Loading page rendering', () => {
    it('loading page renders correctly', () => {
      expect(renderer.create(<LoadingPage />).toJSON()).toMatchSnapshot()
    })
  });