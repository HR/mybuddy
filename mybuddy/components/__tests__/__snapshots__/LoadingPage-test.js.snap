// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading page rendering loading page renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
        "height": 1334,
      },
      Object {
        "justifyContent": "center",
      },
    ]
  }
>
  <ActivityIndicator
    animating={true}
    color="#EA952F"
    hidesWhenStopped={true}
    size="large"
    style={
      Object {
        "height": 80,
      }
    }
  />
</View>
`;
